project_stage_closed/models/__init__.py

```diff
@@ -1,2 +1,4 @@
+# Copyright 2018 Tecnativa - Ernesto Tejeda
```

Comment:
Please avoid to add the copyright line in the `__init__.py`, as stated in the guidelines